proto/v1beta1/grafeas.proto

```diff
@@ -34,6 +34,7 @@ import "proto/v1beta1/image.proto";
 import "proto/v1beta1/package.proto";
 import "proto/v1beta1/provenance.proto";
 import "proto/v1beta1/vulnerability.proto";
+import "proto/v1beta1/intoto.proto";
```

Comment:
A good convention to follow is to sort the import statements.
See https://developers.google.com/protocol-buffers/docs/style

Reply:
Fixed. :)